modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -128,8 +128,11 @@ if(CONFIG_NRFX_TWI OR CONFIG_NRFX_TWIM)
   zephyr_library_sources(${SRC_DIR}/nrfx_twi_twim.c)
 endif()
 
-if (CONFIG_NRF_GRTC_TIMER AND CONFIG_NRF_GRTC_TIMER_CLOCK_MANAGEMENT)
-  zephyr_compile_definitions(NRF_GRTC_HAS_EXTENDED=1)
+if (CONFIG_NRF_GRTC_TIMER)
+  zephyr_compile_definitions_ifdef(CONFIG_NRF_GRTC_TIMER_CLOCK_MANAGEMENT NRF_GRTC_HAS_EXTENDED=1)
+  zephyr_compile_definitions_ifdef(CONFIG_NRF_GRTC_SLEEP_ALLOWED NRFX_GRTC_CONFIG_SLEEP_ALLOWED=1)
+  zephyr_compile_definitions_ifdef(CONFIG_NRF_GRTC_TIMER_AUTO_KEEP_ALIVE NRFX_GRTC_CONFIG_AUTOEN=1)
+  zephyr_compile_definitions_ifdef(CONFIG_NRF_GRTC_START_SYSCOUNTER NRFX_GRTC_CONFIG_AUTOSTART=1)
```

Comment:
Why not do this translation in `nrfx_config.h` as we do with other Kconfig options?

Reply:
Is that ok?
https://github.com/zephyrproject-rtos/zephyr/pull/71688/files#diff-a9116ec095279ea70645bb20cd98c5b8be1c82da1c6b1d199ea77662aa2ccdbdR74-R91

Reply:
Almost.
Please locate these entries right after the already existing GRTC-related ones: https://github.com/zephyrproject-rtos/zephyr/blob/84fd3d5db1797c1e8b4fef8671bf3e86fa24be11/modules/hal_nordic/nrfx/nrfx_config.h#L134-L139 and get rid of the redundant `#ifdef CONFIG_NRF_GRTC_TIMER` (other `CONFIG_NRF_GRTC_*` symbols are anyway not defined when this one is not). Also the blank lines between these added entries are unnecessary.

Reply:
done